docs/make.jl

```diff
@@ -1,58 +1,32 @@
-using Documenter
+using Documenter, DocumenterMarkdown
 using DataFrames
 using CategoricalArrays
 
-DocMeta.setdocmeta!(DataFrames, :DocTestSetup, :(using DataFrames); recursive=true)
-
 # Build documentation.
 # ====================
-
 makedocs(
-    # options
     modules = [DataFrames],
-    doctest = true,
-    clean = false,
-    sitename = "DataFrames.jl",
-    format = Documenter.HTML(
-        canonical = "https://juliadata.github.io/DataFrames.jl/stable/",
-        assets = ["assets/favicon.ico"],
-        edit_link = "main"
-    ),
-    pages = Any[
-        "Introduction" => "index.md",
-        "First Steps with DataFrames.jl" => "man/basics.md",
-        "User Guide" => Any[
-            "Getting Started" => "man/getting_started.md",
-            "Working with DataFrames" => "man/working_with_dataframes.md",
-            "Importing and Exporting Data (I/O)" => "man/importing_and_exporting.md",
-            "Joins" => "man/joins.md",
-            "Split-apply-combine" => "man/split_apply_combine.md",
-            "Reshaping" => "man/reshaping_and_pivoting.md",
-            "Sorting" => "man/sorting.md",
-            "Categorical Data" => "man/categorical.md",
-            "Missing Data" => "man/missing.md",
-            "Data manipulation frameworks" => "man/querying_frameworks.md",
-            "Comparison with Python/R/Stata" => "man/comparisons.md"
-        ],
-        "API" => Any[
-            "Types" => "lib/types.md",
-            "Functions" => "lib/functions.md",
-            "Indexing" => "lib/indexing.md",
-            "Metadata" => "lib/metadata.md",
-            hide("Internals" => "lib/internals.md"),
-        ]
-    ],
-    strict = true
+    clean=true,
+    doctest=false,
+    sitename="DataFrames.jl",
+    authors="Bogumił Kamiński et al.",
```

Comment:
@lazarusA - thank you for putting this 😄, but we need some wider agreement on the authors. @nalimilan - what do you think we should have here? Maybe `JuliaData et at.` (or indeed putting some names?)